### PR TITLE
Replace user_id with distinct_id

### PR DESCRIPTION
**Pull request by ark-konopacki**
related to https://github.com/calabash/calabash-ios/issues/985

### CODE REVIEW

**Comment by jmoody** [comment]
Jenkins test this please

File: calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb [hunk 1]
```diff
@@ -163,7 +168,7 @@ def info
         hash = {
           :event_name => "session",
           :data_version => DATA_VERSION,
-          :user_id => user_id
+          :distinct_id => distinct_id
```
Comment (jmoody) [inline calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb]: Sadly, I think the only change that needed to be made was:

```
:distinct_id => user_id
```

**Comment by ark-konopacki** [comment]
@jmoody can you please check?
I try locally `be rspec calabash-ios/calabash-cucumber/spec/integration/console_helpers_spec.rb` and it passed.

**Comment by jmoody** [comment]
Console rspec tests are failing.  Will fix in a subsequent pull request.

**Pull request merged**